BaseFilter/sections/foreign.txt

```diff
@@ -608,6 +608,7 @@ gerasanews.com##.iads_border
 qatarshares.com##.above_body > div#header+table[width="100%"][cellpadding="0"]
 adslgate.com##.smallfont > center > a > img
 adslgate.com##.smallfont a[href^="http://deals.souq.com/"]
+adslgate.com##.skyscraper-banner-right
```

Comment:
Could you please move the rule next to the comment `! NOTE: Arabian` with the task link?
Like
```adblock
! https://github.com/AdguardTeam/AdguardFilters/issues/176591
adslgate.com##.skyscraper-banner-right
```

Reply:
Please also check if the current rules are obsolete or not.

Reply:
I think it is not a problem - the rule is not dangerous.

Reply:
Ok.

Reply:
Reviewing of old rules is also not a task of this event)